app/src/main/java/com/kamron/pogoiv/updater/AppUpdateLoader.java

```diff
@@ -27,10 +26,19 @@ public void run() {
         try {
             Response response = httpClient.newCall(request).execute();
             JSONArray releaseInfo = new JSONArray(response.body().string());
-            JSONObject latestRelease = releaseInfo.getJSONObject(0);
-            JSONObject releaseAssets = latestRelease.getJSONArray("assets").getJSONObject(0);
 
-            AppUpdate update = new AppUpdate(releaseAssets.getString("browser_download_url"), latestRelease.getString("tag_name"), latestRelease.getString("body"));
+            //Loop through the releases till we get to the first online release
+            JSONObject latestOnlineRelease = releaseInfo.getJSONObject(0);
+            for(int i = 0;i < releaseInfo.length();i++){
+                JSONObject latestRelease = releaseInfo.getJSONObject(i);
+                if(!latestRelease.getString("tag_name").contains("(Offline)")) {
```

Comment:
This is a Git tag_name—and the tag name for online and offline releases would usually be the same (since they're based off the same sources).
Looking at https://api.github.com/repos/farkam135/GoIV/releases, it'd be easier to look at the assect name and check whether it contains `-Online` (like `GoIV-Online.2.2.0.apk`) or not.